src/compas/colors/colordict.py

```diff
@@ -0,0 +1,95 @@
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import division
+
+from compas.data import Data
+from .color import Color
+
+
+class ColorDict(Data):
```

Comment:
is it worth it to inherit from `UserDict` instead of implementing our own dict logic? Not a lot, but it would shave off a few lines of code that can be used directly from that class